cfgov/v1/models/browse_page.py

```diff
@@ -35,6 +36,7 @@ class BrowsePage(CFGOVPage):
         ('table_block', organisms.AtomicTableBlock(table_options={'renderer':'html'})),
         ('job_listing_table', JobListingTable()),
         ('feedback', v1_blocks.Feedback()),
+        ('conference_registration_form', ConferenceRegistrationForm()),
```

Comment:
I think because `ConferenceRegistrationForm` was created as a block under `data_research.blocks` instead of under `v1.blocks`.